book/src/developers/contributing/build_instructions/linux.md

```diff
@@ -187,7 +187,7 @@ variables during testing.
 ```sh
 $ cargo test --workspace
 $ cargo run -- --discovery-port 9009 \
-    --web3-http-address 127.0.0.1:8547 \
+    --web3-http-address 127.0.0.1:8545 \
```

Comment:
This is the default listening address here. IIUC in this example, we're trying to show how to start a second trin instance alongside an already-running trin instance, so this would cause an address conflict. And since we're using `8546` as websocket, I'd say we can just leave this port as `8547`

Reply:
As discussed, I added clarifications.

Note for others: All situation regarding ports is a bit confusing, especially considering we can't change the websocket port at the moment. Will create an issue to follow up on this.